src/components/TodayScreen/TodayScreen.js

```diff
@@ -32,14 +33,21 @@ export default function TodayScreen() {
             promise.then(response => {
                 setControl(true);
                 setCounter(counter + 1);
-            })
+            });
         }
         if (done === true) {
```

Comment:
**suggestion**: this if could just be an else, since we only have two possible scenarios here, WDYT?